src/options.rs

```diff
@@ -114,6 +114,9 @@ pub struct HyperfineOptions {
     /// Whether or not to ignore non-zero exit codes
     pub failure_action: CmdFailureAction,
 
+    /// Command to run before each batch of timing runs
+    pub setup_command: Option<Vec<String>>,
```

Comment:
This should be just a `Option<String>`, since only one setup command can be provided.